mixins/localize-mixin.md

```diff
@@ -121,6 +121,49 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method
+
+### Simple Markup
+
+#### `[b]Bold[/b]`
+The `[b]` markup converts to the `<strong>` tag
+
+#### `[i]Italic[/i]`
+The `[i]` markup converts to the `<em>` tag
+
+#### `[br]`
+The `[br]` markup converts to the `<br />` tag
+
+
+### Complex markup
+
+Complex markup is more powerful, but requires passing additional information in the second parameter
+
+#### `[a]Linked[/a]`
+The `[a]` markup converts to the `<d2l-link>` tag, and takes an attributes string in the `_links` property
```

Comment:
Wondering if this should be `[link]` in order to align better with SML? 🤔

Reply:
I've gone back and forth on it, because

1. It's not the same as the SML. Customers really shouldn't be using any of these, as they can't know if `localizeHTML` is being called, and even if they try it on one that is, links and tooltips (and html) can't be provided with the data they need from the lang term (nor should they be, I think).
2. I was trying to keep everything as short as possible for readability/translators, but maybe having a useful name adds enough context to actually be more helpful.

Reply:
Agreed that since it's not the same as SML, we shouldn't feel obliged to align with it... and yeah maybe even purposefully distance ourselves from it to avoid confusion. I also agree that the data links/tooltips/etc. need shouldn't come from the lang term at all.